content/en/news/releases/1.7.x/announcing-1.7.4/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Announcing Istio 1.7.4
+linktitle: 1.7.4
+subtitle: Patch Release
+description: Istio 1.7.4 patch release.
+publishdate: 2020-10-27
+release: 1.7.4
+aliases:
+    - /news/announcing-1.7.4
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.7.3 and Istio 1.7.4
+
+{{< relnote >}}
+
+## Changes
+
+- **Improved** TLS configuration on sidecar server side inbound paths to enforce TLS 2.0 version along with recommended cipher suites. This is disabled by default and can enabled by setting env variable `PILOT_SIDECAR_ENABLE_INBOUND_TLS_V2` to true.
+
+- **Added** ability to configure domain suffix for multicluster installation (Issue #27300)
```

Comment:
the issues should probably be links?

Reply:
Nice catch! I have added links to each note. The recent releasenotes do not have links or issue ids. Maybe we can add a format check to enforce this.

Reply:
Nice catch! Thanks! I have added the links manually. I also filed https://github.com/istio/istio/issues/28276, a format check on release notes might be helpful.